src/components/TextField/style.css

```diff
@@ -0,0 +1,291 @@
+}
+
+.standard-input:focus ~ .standard-input-label,
+.standard-input:valid ~ .standard-input-label {
+  top: -15px;
+  font-size: 14px;
+  color: #1976d2;
+}
+
+.standard-input:focus ~ .standard-input-label-err,
+.standard-input:valid ~ .standard-input-label-err {
+  top: -15px;
+  font-size: 14px;
+  color: #f44336;
+}
+
+.standard-input-bar {
+  position: relative;
+  display: block;
+  width: 300px !important;
+}
+
+.standard-input-bar-err {
+  position: relative;
+  display: block;
+  width: 300px !important;
+}
+
+.standard-input-bar:before,
+.standard-input-bar:after {
+  content: '';
+  height: 2px;
+  width: 0;
+  bottom: -0.5px;
+  position: absolute;
+  background: #1976d2;
+  transition: 0.2s ease all;
+}
+
+.standard-input-bar-err:before,
+.standard-input-bar-err:after {
+  content: '';
+  height: 2px;
+  width: 0;
+  bottom: -0.5px;
+  position: absolute;
+  background: #f44336;
+  transition: 0.2s ease all;
+}
+
+.standard-input-bar:before {
+  left: 50%;
+}
+
+.standard-input-bar-err:before {
+  left: 50%;
+}
+.standard-input-bar:after {
+  right: 50%;
+}
+
+.standard-input-bar-err:after {
+  right: 50%;
+}
+
+.standard-input:focus ~ .standard-input-bar:before,
+.standard-input:focus ~ .standard-input-bar:after {
+  width: 50% !important;
+}
+
+.standard-input:focus ~ .standard-input-bar-err:before,
+.standard-input:focus ~ .standard-input-bar-err:after {
+  width: 50% !important;
+}
+/**** standard input ****/
+
+/**** filled input ****/
+.filled-input {
+  background: #e8e8e8;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  font-size: 18px;
+  padding: 10px 10px 10px 10px !important;
+  display: block;
+  width: 300px;
+  border: none;
+  border-bottom: 1px solid #757575;
+  transition: all 400ms;
+}
+
+.filled-input:hover {
+  background: #dedede !important;
+}
+
+.filled-input:focus {
+  outline: none;
+}
+
+.filled-input-label,
+.filled-input-label-err {
+  color: #999;
+  font-size: 18px;
+  font-weight: normal;
+  position: absolute;
+  pointer-events: none;
+  left: 10px;
+  top: 13px;
+  transition: 0.2s ease all;
+}
+
+.filled-input:focus ~ .filled-input-label,
+.filled-input:valid ~ .filled-input-label {
+  top: -20px;
+  font-size: 14px;
+  color: #1976d2;
+}
+
+.filled-input:focus ~ .filled-input-label-err,
+.filled-input:valid ~ .filled-input-label-err {
+  top: -20px;
+  font-size: 14px;
+  color: #f44336;
+}
+
+.filled-input-bar {
+  position: relative;
+  display: block;
+  width: 300px;
+}
+
+.filled-input-bar-err {
+  position: relative;
+  display: block;
+  width: 300px;
+}
+
+.filled-input-bar:before,
+.filled-input-bar:after {
+  content: '';
+  height: 2px;
+  width: 0;
+  bottom: -0.5px;
+  position: absolute;
+  background: #1976d2;
+  transition: 0.2s ease all;
+}
+
+.filled-input-bar-err:before,
+.filled-input-bar-err:after {
+  content: '';
+  height: 2px;
+  width: 0;
+  bottom: -0.5px;
+  position: absolute;
+  background: #f44336;
+  transition: 0.2s ease all;
+}
```

Comment:
include both the below classes in upper code, remove same code just let the background property remain which is different.